main.py

```diff
@@ -1,10 +1,53 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Guild.objects.all().delete()
+    Player.objects.all().delete()
```

Comment:
There is no need for this part, but in case you added it you could omit `all()`
```suggestion
    Race.objects.delete()
    Skill.objects.delete()
    Guild.objects.delete()
    Player.objects.delete()
```